connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/connect/client/GrpcExceptionConverter.scala

```diff
@@ -47,11 +52,37 @@ private[client] object GrpcExceptionConverter {
     }
   }
 
-  private def toSparkThrowable(ex: StatusRuntimeException): SparkThrowable with Throwable = {
+  private val errorFactory = new ErrorFactoryBuilder()
+    .addConstructor((message, _) => new ParseException(None, message, Origin(), Origin()))
```

Comment:
How do we plan to pass other information here? like `start` and `stop`

Reply:
The current plan is to only construct exceptions by (message, cause) on the client side. We may need to extend Spark related exceptions to support (message, cause) constructor if they don't have one